RaveBusinessLogic/pyBRAT.xml

```diff
@@ -3,46 +3,193 @@
   <!-- <Project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="./XSD/project_explorer.xsd"> -->
   <Name>pyBRAT</Name>
   <ProjectType>pyBRAT</ProjectType>
-  <Node xpathlabel="Name" xpath="Realizations/pyBRAT">
+  <Node xpathlabel="Name">
     <Children collapsed="false">
-      <Node label="Outputs">
+      <Node label="Outputs" xpath="Realizations/pyBRAT/Analyses/Analysis/Outputs">
         <Children collapsed="true">
+          <Node xpathlabel="Name" xpath="SQLiteDB" type="file" />
+          <Node xpathlabel="Name" xpath="HTMLFile" type="report" />
           <Node label="Capacity">
             <Children>
-              <Node label="Existing Dam Capacity" xpath="Outputs/Vector[@id='BRAT_TABLE']" type="line" symbology="oCC_EX" />
-              <Node label="Historic Dam Capacity" xpath="Outputs/Vector[@id='BRAT_TABLE']" type="line" symbology="oCC_HPE" />
+              <Node label="Existing Dam Capacity" xpath="Vector[@id='COMBINED_CAPACITY']" type="line" symbology="oCC_EX" id="ex_capacity" />
+              <Node label="Existing Dam Size" xpath="Vector[@id='COMBINED_CAPACITY']" type="line" symbology="mCC_EX_CT" />
+              <Node label="Historic Dam Capacity" xpath="Vector[@id='COMBINED_CAPACITY']" type="line" symbology="oCC_HPE" id ="his_capacity"/>
+              <Node label="Historic Dam Size" xpath="Vector[@id='COMBINED_CAPACITY']" type="line" symbology="mCC_HPE_CT" />
             </Children>
           </Node>
           <Node label="Management">
             <Children collapsed="true">
-              <Node label="Restoration or Conservation Opportunities" xpath="Outputs/Vector[@id='BRAT_TABLE']" type="line" symbology="oPBRC_CR" />
+              <Node label="Unsuitable or Limited Opportunities" xpath="Vector[@id='OPPORTUNITY']" type="line" symbology="oPBRC_UD" id="limited" />
+              <Node label="Risk of Undesirable Dams" xpath="Vector[@id='RISK']" type="line" symbology="oPBRC_UI" id="risk" />
+              <Node label="Restoration or Conservation Opportunities" xpath="Vector[@id='OPPORTUNITY']" type="line" symbology="oPBRC_CR" id="opportunity" />
```

Comment:
I am confused why these are showing up as new when they were correct in original that I referenced above? 
https://github.com/Riverscapes/RiverscapesXML/blob/691108896b92158cea56ae9f67cfbdb3112f9922/RaveBusinessLogic/pyBRAT.xml#L23